api/Navigator.json

```diff
@@ -3411,15 +3411,15 @@
           "spec_url": "https://w3c.github.io/badging/#setappbadge-method",
           "support": {
             "chrome": {
-              "version_added": "83",
+              "version_added": "81",
               "partial_implementation": true,
               "notes": "Windows and macOS only."
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": "81"
```

Comment:
Is this based on testing Chrome for Android? It wouldn't be surprising if this API was shipped at a different time on desktop platforms and Android.

Reply:
This was, yes -- the rest of the Android browsers were mirrored, but testing those in their latest versions showed support.